packages/@vue/cli-plugin-unit-jest/generator/index.js

```diff
@@ -12,7 +12,10 @@ module.exports = (api, options, rootOptions, invoking) => {
       'test:unit': 'vue-cli-service test:unit'
     },
     devDependencies: {
-      'vue-jest': isVue3 ? '^5.0.0-0' : '^4.0.1',
+      'babel-jest': '^27.0.6',
+      'jest': '^27.0.5',
+      '@vue/vue2-jest': isVue3 ? undefined : '^27.0.0-alpha.1',
```

Comment:
The package is temporarily published as `@vue/vue2-jest`, as someone squatted `vue2-jest`. @lmiller1990 is in the process of reclaiming it from NPM.

Reply:
I emailed a few times but npm don't seem to respond. Maybe we need Evan to reach out.

Alternatively, we can just go with `@vue/vue2-jest` and `@vue/vue3-jest`... it seems pretty explicit. What do you think?

Reply:
Fine by me. `vue-jest` is ready since PR https://github.com/vuejs/vue-jest/pull/375, so we just need a release for `@vue/vue3-jest` I think

Reply:
Fine by me, too. Using scoped names also helps avoid similar dilemmas when we have Vue 4…

Reply:
I will release a new vue-jest within 24 of this post.

Reply:
It took 48 hours but see here https://github.com/vuejs/vue-jest/releases/tag/v27.0.0-alpha.1

- `@vue/vue2-jest` 
- `@vue/vue3-jest`

Vue Jest versioning is still really rough - one of them is at alpha.1, and the other alpha.2. Anyway, hopefully this helps, we can push out another version if needed. I like the new versioning scheme - it's easy to understand.